.github/workflows/docker.yaml

```diff
@@ -0,0 +1,100 @@
+name: Build and publish docker image on new tag creation
+on:
+  push:
+    tags:
+      - release.*
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  # Label of the build job
+  build:
+    # Containers must run in Linux based operating systems
+    runs-on: ubuntu-22.04
+    # Docker Hub image that `container-job` executes in
+    container: node:20.10
+    steps:
+      - name: install node v18
```

Comment:
nitpick: I would change it to `install node`, as if we specify the version we need to keep it aligned to the actual version of node used